src/pages/landing/ebooks/forex.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import Introduction from 'components/layout/ebooks/introduction'
+import HeaderSection from 'components/layout/ebooks/header'
+import Topics from 'components/layout/ebooks/topics'
+import Layout from 'components/layout/layout'
+import { SEO } from 'components/containers'
+import { localize, WithIntl } from 'components/localization'
+import introForexEbook from 'images/common/ebooks/introduction-forex-ebook.png'
+import forexEbookImage from 'images/common/ebooks/forex-ebook.png'
+import forexEbookInside from 'images/common/ebooks/forex-ebook-inside.png'
```

Comment:
Let's query this using graphql. You can refer to this ref on why we do it. 
https://www.gatsbyjs.com/docs/working-with-images/#:~:text=Querying%20images%20with%20GraphQL%20allows,high%2Dperformance%20image%20processing%20library.&text=gatsby%2Dtransformer%2Dsharp%20allows%20you,and%20resolutions%20with%20a%20query